### ISSUE TITLE
Add big picture mode

### ISSUE THREAD

**Issue by picsi** [omarroth/invidious#33]
It'd be very useful to have an enlarged resolution, similar to how Hooktube does it, which can be toggled on and off in every video page.

**Comment by dimqua** [comment]
It's too big for my screen, I use:
```
.pure-u-md-16-24 {
    width: 1280px !important;
    margin-left: auto;
    margin-right: auto;
}
```
Also, invidious does not support 1080p videos yet, so I don't see any reason to upscaling videos too much.

**Comment by dotdapple** [comment]
Been using Invidious today and I’m spontaneously getting a wider video player – 83.33% width instead of the usual 66.67% width – for some videos. Really liking it.

**Comment by omarroth** [comment]
Haha, sorry about that. Doing a migration for #154 and it's taking a bit longer than I expected.

**Comment by omarroth** [comment]
I think I'll consider this closed with e0ee1c3d79f51fa77e30718425a10411263a6b3e. There hasn't been much activity in this issue, but I am very much interested in improving the UI. For things like this I'd recommend opening a new issue or a PR with some proposed changes.
